rpc/client/peer-rpc-clnt.go

```diff
@@ -49,3 +49,34 @@ func ValidateAddPeer(p *peer.PeerAddRequest) (*services.RPCPeerAddResp, error) {
 	}
 	return rsp, nil
 }
+
+func ExportEtcdEnvVar(argv *services.RPCEtcdEnvReq) (*services.RPCEtcdEnvResp, error) {
```

Comment:
calling of this missed in this patch. i have already mentioned  in my previous PR that i need to do some modification in this patch.